src/app/features/ui/filter-search-button/filter-search-button.component.scss

```diff
@@ -0,0 +1,103 @@
+@use './src/styles/colors' as st;
+
+.container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: max-content;
+}
+.filter-btn {
+  border: none;
+  padding: 2.2rem;
+  cursor: pointer;
+  border-radius: 100px;
+  background-image: url('/assets/icons/filter-inactive.svg');
+  background-position: center;
+  background-size: cover;
+  margin-bottom: 0.5rem;
+}
+
+.filter-btn--active {
+  background-image: url('/assets/icons/filter-active.svg');
+}
+
+.dropdown {
+  background-color: st.$primary-900;
+  color: st.$neutral-600;
+  display: flex;
+  justify-content: space-evenly;
+  padding: 1rem clamp(0.5rem, 3vw + 0.4rem, 2.5rem) clamp(0.5rem, 4vw + 0.4rem, 4rem)
+    clamp(0.5rem, 3vh + 0.5rem, 2.5rem);
+  border-radius: 15px;
+  flex-wrap: wrap;
+  width: max-content;
+  column-gap: 2rem;
+  &__section {
+    border: none;
+    display: flex;
+    flex-direction: column;
+
+    border: none;
+    &:last-child {
+      p {
+        width: 130px;
+      }
+    }
+    p {
+      color: st.$neutral-100;
+      padding: 0.5rem 0;
+      border-bottom: 1px solid st.$neutral-500;
+      min-width: max-content;
+    }
+    div:nth-child(2) {
+      margin-top: 0.5rem;
+    }
+    &__holder {
+      padding: 0.1rem 0;
+      position: relative;
+      label {
+        width: 80%;
+        display: inline-block;
+        cursor: pointer;
+      }
+
+      &__filtr-option {
+        border: none;
+        display: none;
+      }
+      &__filtr-option:checked {
+        border: none;
+      }
+
+      &:has(.dropdown__section__holder__filtr-option:checked) {
+        color: st.$neutral-100;
+        display: inline-block;
+      }
+      &:has(.dropdown__section__holder__filtr-option:checked)::after {
+        content: '\0078';
+        color: st.$neutral-100;
+        cursor: pointer;
+      }
+      &__sort-option {
+        border: none;
+
+        display: none;
+      }
+      &:has(.dropdown__section__holder__sort-option:checked) {
+        color: st.$neutral-100;
+        display: inline-block;
+      }
+    }
+  }
+}
+
+.dropdown--unit {
+  justify-content: flex-start;
+
```

Comment:
pusta linia

Reply:
Zostawiam sobie czasem wolne miejsca aby zwiększyć czytelność, w tym przypadku jest to po to aby było bardziej widoczne że ta klasa nie jest zagnieżdżona wewnątrz żadnej innej. Podczas budowania aplikacji kompilator i tak usuwa puste linie, więc to nie ma wpływu na wydajność.